doc/reference/networking/sockets.rst

```diff
@@ -0,0 +1,8 @@
+Socket Options
+==============
+
+IPv4 Options
+------------
+
+- ``IP_MULTICAST_LOOP``: Controls whether multicast packets are looped back to
+  the sending host. Set to 1 to enable loopback, 0 to disable.
```

Comment:
This file is not really needed, or if we want to have it it should contain documentation of all the options and not just for this one. Anyway, that is something for another PR.